Extension/src/LanguageServer/client.ts

```diff
@@ -127,6 +128,16 @@ function showMessageWindow(params: ShowMessageWindowParams): void {
     }
 }
 
+function showWarning(params: ShowWarningParams): void {
+    const message: string = util.getLocalizedString(params.localizeStringParams);
+    if (!warningChannel) {
+        warningChannel = vscode.window.createOutputChannel(`${localize("c.cpp.warnings", "C/C++ Warnings")}`);
```

Comment:
Is C/C++ Warnings too vague? It could imply IntelliSense warnings (which aren't enabled yet). Would using C/C++ Configuration Warnings be more clear?